src/main/java/com/google/sps/servlets/RecipeServlet.java

```diff
@@ -127,25 +137,22 @@ private String getRecipeList() {
     return gson.toJson(recipeList);
   }
 
+  private Query recipeWhereContainsArray(String[] tagIDs, int index) {
```

Comment:
Difficult to figure out exactly what this method suppose to do. Mind adding a comment?

Reply:
Will do, also planning to rename for clarity